src/main/java/alison/task/Task.java

```diff
@@ -0,0 +1,42 @@
+package alison.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
I think it is good naming of the boolean variable as `isDone` with the emphasis instead of `done`!